src/ui/opengl/castlecontrols_edit.inc

```diff
@@ -361,4 +362,16 @@ begin
     Result := inherited CapturesEventsAtPosition(Position);
 end;
 
+procedure TCastleEdit.SetFocused(const Value: boolean);
+begin
+  inherited;
+
+  {$ifdef ANDROID}
```

Comment:
Same as above -- simply do `Messaging.Send` always, not only on Android, so no need for `{$ifdef ANDROID}`. This way it will also be useful on iOS.